code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -426,7 +426,14 @@
 	icon_state = "trappistbottle"
 	volume = 50
 	list_reagents = list(/datum/reagent/consumable/ethanol/trappist = 50)
-
+	
+// hippie start -- New bottled version of Nuka Cola for use in Cola vending machines
+/obj/item/reagent_containers/food/drinks/bottle/nukacola
+	name = "Nuka Cola"
+	desc = "Don't cry, Don't raise your eye, it's only nuclear wasteland."
+	icon_state = "nuka_colaglass"
+	list_reagents = list(/datum/reagent/consumable/nuka_cola = 50)
+// hippie end
```

Comment:
Remove the hippie start and hippie end, its not a matter of giving credit or not, its just that we dont need those markers

Reply:
*gasp